dev-python/xdg/xdg-5.0.1.ebuild

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7..9} )
+inherit distutils-r1
+
+DESCRIPTION="Implementation of the XDG Base Directory Specification in Python"
+HOMEPAGE="https://pypi.org/project/xdg"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="ISC"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
```

Comment:
Tests are a single file right, so not worth bothering, or?

Reply:
Yeah, it's a single file, also, I tested github tarball, had to use pyproject.toml which apparantly doesn't work? (`file xdg.py (for module xdg) not found` so I just did a build from pypi tarball for now (it has a setup.py)